src/sentry/issues/group_policy.py

```diff
@@ -0,0 +1,167 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import TYPE_CHECKING, Any, Dict, Optional, Set, Type
+
+from django.conf import settings
+
+from sentry.issues.grouptype import GroupType
+from sentry.models import Organization, Project
+from sentry.utils import metrics, redis
+
+if TYPE_CHECKING:
+    from sentry.utils.performance_issues.performance_detection import PerformanceProblem
+
+_group_policy_registry: Dict[int, Type[GroupPolicy]] = {}
+DEFAULT_IGNORE_LIMIT: int = 3
+DEFAULT_EXPIRY_TIME: int = 60 * 60 * 24
+
+
+@dataclass(frozen=True)
+class NoiseConfig:
+    ignore_limit: int = DEFAULT_IGNORE_LIMIT
```

Comment:
Will `ignore_limit` be removed from the `GroupType` dataclass?

Reply:
yeah in the next PR presumably. if you write any new docs or guidelines I will update them